CMakeLists.txt

```diff
@@ -77,7 +76,7 @@ if(DEFINED ENV{ANALYZE})
 endif()
 
 # Finished setting compiler/compiler flags.
-set(CMAKE_CXX_FLAGS "${C_COMPILE_FLAGS} ${CXX_COMPILE_FLAGS}"
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CXX_COMPILE_FLAGS}"
```

Comment:
This previously did not include `CMAKE_CXX_FLAGS` meaning we'd wipe out any externally-set flags.